policies/aws.nlb.high.activeflowcount.json

```diff
@@ -0,0 +1,31 @@
+{
+  "policy": {
+    "actions": [
+      {
+        "category": 3,
+        "type": "event"
+      }
+    ],
+    "conditions": [
+      {
+        "analytic": "baselineDeviation",
+        "metric": "aws.networkelb.activeflowcount",
+        "operator": ">"
+      },
+      {
+        "analytic": "actual",
+        "level": 1000,
+        "metric": "aws.networkelb.activeflowcount",
+        "operator": ">="
+      }
+    ],
+    "deleted": false,
+    "description": "This policy will generate a Critical alert when we detect unusually high Active Flow Counts (connections) with a minimum of 1000 Active Flows for at least 15 minutes.",
```

Comment:
I don't like this policy.  It suggests jumps in connections are only relevant for this with >1000 connections.  Is it really Critical if we get an abnormally high (baselineDeviation) amount of connections if everything else is fine?  Which we can't answer so at best this is Warning, probably Info.